src/types/pick.ts

```diff
@@ -0,0 +1,103 @@
+import { BaseObjectLikeTypeImpl, createType } from '../base-type.js';
+import type {
+    BasicType,
+    MessageDetails,
+    OneOrMore,
+    PossibleDiscriminator,
+    Properties,
+    PropertiesInfo,
+    Result,
+    Type,
+    TypeImpl,
+    TypeOf,
+    ValidationOptions,
+    Visitor,
+} from '../interfaces.js';
+import { decodeOptionalName } from '../utils/collection-utils.js';
+import { define } from '../utils/define.js';
+import { unknownRecord } from './unknown.js';
+import { hasOwnProperty } from '../utils/type-utils.js';
+import { prependPathToDetails } from '../utils/failure-utils.js';
+import { interfaceStringify } from '../utils/stringifiers.js';
+import { propsInfoToProps } from './union.js';
```

Comment:
This probably happened because of the setting I mentioned earlier. Should be more like this: (no `.js` suffix and using commonjs-style barrel imports, e.g. `'../utils'`)
```suggestion
import { BaseObjectLikeTypeImpl, createType } from '../base-type';
import type {
    BasicType,
    MessageDetails,
    OneOrMore,
    PossibleDiscriminator,
    Properties,
    PropertiesInfo,
    Result,
    Type,
    TypeImpl,
    TypeOf,
    ValidationOptions,
    Visitor,
} from '../interfaces';
import { decodeOptionalName, define, hasOwnProperty, interfaceStringify, prependPathToDetails } from '../utils';
import { propsInfoToProps } from './union';
import { unknownRecord } from './unknown';
```